### PR TITLE
Select Field

### PR DESCRIPTION
**Pull request by jasonfellows**
Add a wrapper component for Simple Select.

### CODE REVIEW

File: src/js/components/forms/fields/select/select-field.js [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+  },
+
+  getInitialState() {
+    return {
+      disabled: this.props.disabled || false,
+      errors: this.props.errors || [],
+    };
+  },
+
+  borderColor() {
```
Comment (HipsterBrown) [inline src/js/components/forms/fields/select/select-field.js]: Where does this get used in this file?

Comment (jasonfellows) [inline src/js/components/forms/fields/select/select-field.js]: Nowhere, removed

File: src/js/components/forms/fields/select/select-field.js [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+  },
+
+  handleChange(e) {
+    this.props.onChange();
+    this.resetErrors();
+  },
+
+  label() {
+    if (this.props.label) {
+      return <label className="px2 mb1 relative">
```
Comment (HipsterBrown) [inline src/js/components/forms/fields/select/select-field.js]: Instead:

```js
return (
   <label></label>
)
```

**Comment by tsargent** [comment]
+1

**Pull request merged**